jobs/refresh-functions-n-views.yml

```diff
@@ -0,0 +1,12 @@
+# The purpose of this job is to refresh views and functions
```

Comment:
I don't understand this job file.  We already have a job whose purpose is to refresh source views and functions - it's the "create-source-views-and-functions.yml" referenced below.  I don't know what the purpose of this new job is and why it includes the iterating-job that uses "import-to-table-partition.yml" job.  Can you clarify why you created this?